### DISCUSSION TITLE
Can't install on Mac

### DISCUSSION THREAD

**Discussion by songproducer** [searx/searx#1152] [Installation issue: custom]
When I run:
./manage.sh update_packages

I get:

> readlink: illegal option -- f
> usage: readlink [-n] [file ...]
> ./manage.sh: line 11: return: can only `return' from a function or sourced script
> Requirement already up-to-date: pip in ./searx-ve/lib/python2.7/site-packages
> Requirement already up-to-date: setuptools in ./searx-ve/lib/python2.7/site-packages
> Could not open requirements file: [Errno 2] No such file or directory: '/requirements.txt'

Even though there is a requirements.txt file in the directory.

**Comment by dalf** [comment]
About readlink on mac os :
https://stackoverflow.com/questions/1055671/how-can-i-get-the-behavior-of-gnus-readlink-f-on-a-mac

**Comment by HongliYu** [comment]
Check the branch named fix-manage-sh @songproducer

**Comment by RedSoxFan04** [comment]
I don't know what the OP is doing differently, but it works fine for me in Firefox 58 on Mac OS Sierra.

**Comment by unixfox** [comment]
@songproducer Do you still have an issue with installing Searx on macOS? If not then could you close this issue?
